Azure/communication-ui-library-ios
# [Bugfix] Removed applying a global layout direction setting and added setting …

## Purpose

* Replaced global layout direction setting to setting it for the needed screens. We don't need to set it globally. Setting it globally will make the navigation items in demo view controller / demo ui view flip position unexpectedly after quitting call composite.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code


Testing Navigation item in UIKitDemoViewController or SwiftUIDemoView may not be apparent. Try adding navigation items in these two screens and follow the test steps.

```
Run the app, choose Swift UI or UI Kit
In Settings, turn on is Right to left, 
Join call
Follow What to Check section below
```


## What to Check
Verify that the following are valid
* Make sure all the call composite screens follows layout direction settings. 
* Make sure Demo screens does not follow layout direction settings

## Other Information
